data/caasp/update.sh

```diff
@@ -8,22 +8,30 @@ set -exuo pipefail
 usage() {
 	echo "Usage: $0
 		[-s $REPO] Setup all nodes with update REPO
+		[-u] Update all nodes
 		[-c] Check that update was applied
+		[-i] Install package
 		[-r] Just reboot cluster test" 1>&2
 	exit 1
 }
 
-while getopts "s:cr" opt; do
+while getopts ":s:c:u:r:i:" opt; do
```

Comment:
`:` marks options that expect an argument. (When you want getopts to expect an argument for an option, just place a : (colon) after the proper option flag). Please fix or point me to documentation saying otherwise.

Reply:
You are right. I misunderstood the documentation.